packages/request-client.js/test/etherscan-mock.ts

```diff
@@ -40,7 +40,7 @@ export default class EtherscanProviderMock {
           to: '0xc12F17Da12cd01a9CDBB216949BA0b41A6Ffc4EB',
           value: BigNumber.from('0x02dd231b00'),
           nonce: 205,
-          data: '0xc19da4923539c37f',
+          data: '0xefce79375b2db9f7',
           creates: null,
```

Comment:
As we use a new contract version, the requestId is modified resulting in change of payment reference.